front-end/src/Speak.tsx

```diff
@@ -1,11 +1,13 @@
 import * as React from "react";
 import { ttsApiKey } from "./environment";
+import { AddAnnouncementContext } from "./host/HostAnnouncementQueue";
 
 export default function Speak(props) {
   const textToSpeak = props.text;
   const textHasBeenSpoken = React.useRef(false);
+  const addAnnouncement = React.useContext(AddAnnouncementContext);
 
-  async function playElevenLabsAudio() {
+  async function createAnnouncementAudio() {
```

Comment:
Rename this, and make it simply _create_ an audio object, but not play it yet. It is instead added to the queue through the global `addAnnouncement` context.